packages/app-project/src/shared/components/CollectionsModal/components/SelectCollection/SelectCollection.spec.js

```diff
@@ -45,23 +43,13 @@ describe('Component > SelectCollection', function () {
     })
 
     it('should be empty by default', function () {
-      expect(select.prop('value')).to.eql({})
+      expect(select.prop('value')).to.eql(undefined)
     })
 
     it('should list collections', function () {
       expect(select.prop('options')).to.equal(collections)
     })
 
-    it('should call the onSearch callback', function () {
-      const searchText = 'Hello'
-      const searchSpy = onSearch.withArgs({
-        favorite: false,
-        current_user_roles: 'owner,collaborator,contributor',
-        search: searchText
-      })
-      select.simulate('search', searchText)
-      expect(searchSpy).to.have.been.calledOnce()
-    })
```

Comment:
This test has caught a couple of bugs I accidentally introduced in #4867, so we need to keep a test that types into the text box and submits a search.